src/components/Dashboard/Search.vue

```diff
@@ -1,142 +1,353 @@
+                            v-model="status"
+                            @change="searchNow()"
+                            hide-details="auto"
+                          ></status-list>
+                        </div>
+                      </th>
+                      <th
+                        scope="folioNumber"
+                        v-if="canShowColumn('folioNumber')"
+                      >
+                        <v-text-field
+                          id="folioNumber"
+                          autocomplete="off"
+                          class="text-input-style "
+                          filled
+                          label="Folio Number"
+                          v-model="folioNumber"
+                          @change="searchNow()"
+                          hide-details="auto"
+                        />
+                      </th>
+                      <th scope="initiator" v-if="canShowColumn('initiator')">
+                        <v-text-field
+                          id="initiator"
+                          autocomplete="off"
+                          class="text-input-style "
+                          filled
+                          label="Initiator"
+                          v-model.trim="initiator"
+                          @change="searchNow()"
+                          hide-details="auto"
+                        />
+                      </th>
+                      <th scope="total" v-if="canShowColumn('total')">
+                        <v-text-field
+                          id="total"
+                          autocomplete="off"
+                          class="text-input-style "
+                          filled
+                          label="Total Amount"
+                          v-model.trim="totalAmount"
+                          @change="searchNow()"
+                          hide-details="auto"
+                        />
+                      </th>
+                      <th class="action" scope="action">
+                        <span
+                          class="clear-filter primary--text cursor-pointer"
+                          v-if="!searchParamsPrecent"
+                          @click="clearFilter"
+                          >Clear Filters<v-icon small color="primary"
+                            >mdi-close</v-icon
+                          ></span
+                        >
+                      </th>
+                    </tr>
+                  </template>
+                  <template v-slot:item="{ item }">
+                    <transition name="slide-fade">
+                      <tr>
+                        <td v-if="canShowColumn('routingSlipNumber')">
+                          {{ item.number }}
+                        </td>
+                        <td v-if="canShowColumn('receiptNumber')">
+                          <!-- if cash show number else - -->
+                          {{
+                            item.paymentAccount &&
```

Comment:
doesn't vue still have null safe optional chaining?

Reply:
it is braking, undefined error. :(